fem/gridfunc.hpp

```diff
@@ -500,49 +500,126 @@ class GridFunction : public Vector
    virtual void ProjectBdrCoefficientTangent(VectorCoefficient &vcoeff,
                                              const Array<int> &bdr_attr);
 
+   /// @brief Returns ||exsol - u_h||_L2 for scalar or vector H1 or L2 elements
+   ///
+   /// The @a elems input variable expects a list of markers:
+   /// an elem marker equal to 1 will compute the L2 error on that element
+   /// an elem marker equal to 0 will not compute the L2 error on that element
+   ///
+   /// @note Quadratures with negative weights (as in some simplex integration
+   ///       rules in MFEM) can produce negative integrals even with
+   ///       non-negative integrands. To avoid returning negative errors this
+   ///       function uses the absolute values of the element-wise integrals.
+   ///       This may lead to results which are not entirely consistent with
+   ///       such integration rules.
    virtual real_t ComputeL2Error(Coefficient *exsol[],
                                  const IntegrationRule *irs[] = NULL,
                                  const Array<int> *elems = NULL) const;
 
-   /// Returns ||grad u_ex - grad u_h||_L2 in element ielem for H1 or L2 elements
+   /// @brief Returns ||grad u_ex - grad u_h||_L2 in element ielem for H1 or L2 elements
```

Comment:
Long line.

Reply:
Indeed it is. I'll try to rework it (and add argument explanations as suggested later)